modules/redux/package.json

```diff
@@ -18,8 +18,8 @@
   "types": "lib/index.d.ts",
   "scripts": {
     "build": "upbin tsc --declaration",
+    "ci-test": "upbin mocha --require ts-node/register 'test/**/*.test.ts' --color always",
```

Comment:
it would be nice to have coverage tests too.
Could you add "upbin nyc" before the script so it will take coverage data as well?
nyc should work once #203 is merged

Reply:
```suggestion
    "ci-test": "upbin nyc upbin mocha --require ts-node/register 'test/**/*.test.ts' --color always",
```